src/cryptoadvance/specter/global_search.py

```diff
@@ -0,0 +1,260 @@
+import os
+import logging
+
+from flask import url_for
+
+logger = logging.getLogger(__name__)
+
+
+class HtmlElement:
```

Comment:
This sounds cool but i don't have an idea from that description. Maybe it's a good candidate for some tests? It looks like it's quite easy to write some tests for it and that would give a much better picture how this works.

Reply:
Preserving the search-term should be easy via storing it in the session?! I try to get out of the way of UI judgement. Don't have a good track record there. @moneymanolis  @b30wulffz what do you think?